src/Diagnostics.RuntimeHost/Services/ResourceService/SiteService.cs

```diff
@@ -36,7 +36,10 @@ public async Task<StackType> GetApplicationStack(string subscriptionId, string r
             DataTable stackTable = null;
             
             try{
-                stackTable = await dp.Kusto.ExecuteQuery(queryTemplate, DataProviderConstants.FakeStampForAnalyticsCluster, operationName: "GetApplicationStack");
+                if (dataProviderContext.Configuration.KustoConfiguration.CloudDomain == KustoDataProviderConfiguration.AzureCloud)
+                {
+                    stackTable = await dp.Kusto.ExecuteQuery(queryTemplate, DataProviderConstants.FakeStampForAnalyticsCluster, operationName: "GetApplicationStack");
+                }
             }catch(Exception ex){
                 //swallow the exception. Since Mooncake does not have an analytics cluster
                 DiagnosticsETWProvider.Instance.LogRuntimeHostHandledException(dataProviderContext.RequestId, "GetApplicationStack", subscriptionId,
```

Comment:
Yea I think a warning would be better. Perhaps we can change the event level for this event since this event is logged only when we swallow the exception

Reply:
I dont think you would be able to change the Level of the same event to warning in this code and error (when its used in Middleware)

I think I added a WarningEvent for runtime host that we can use. not sure though.

Reply:
No this is not the same one used in the Middleware. Its really subtle but this one is the Handled exception event while the one in the middleware is the unhanded exception event